st2actions/st2actions/query/mistral/v2.py

```diff
@@ -134,15 +136,16 @@ def _format_task_result(self, task):
         """
         Format task result to follow the unified workflow result format.
         """
-        result = {}
-
-        result['id'] = task['id']
-        result['name'] = task['name']
-        result['workflow_execution_id'] = task.get('workflow_execution_id', None)
-        result['workflow_name'] = task['workflow_name']
-        result['created_at'] = task.get('created_at', None)
-        result['updated_at'] = task.get('updated_at', None)
-        result['state'] = task.get('state', None)
+        result = {
+            'id': task['id'],
+            'name': task['name'],
+            'workflow_execution_id': task.get('workflow_execution_id', None),
+            'workflow_name': task['workflow_name'],
+            'created_at': task.get('created_at', None),
+            'updated_at': task.get('updated_at', None),
+            'state': task.get('state', None),
+            'state_info': task.get('state_info', None)
```

Comment:
Hm, as mentioned above and to make it a bit more obvious which attributes are Mistral specific maybe we should put `state_info` in `extra` dictionary (result['extra']['state_info']) or similar?